tests/test_get_schema_options.py

```diff
@@ -0,0 +1,7 @@
+from django.test import TestCase
+
+
+class TestSchemaOptions(TestCase):
+
+    def test_get_options(self):
+        pass
```

Comment:
This is necessary/correct?

Reply:
No, I didn't use this test, I'm removing it.